Example/Controllers/MAFirstLaunchController.swift

```diff
@@ -39,6 +58,80 @@ class MAFirstLaunchController: UIViewController {
     }
 }
 
+// MARK: - UITableViewControllerDelegate
+extension MAFirstLaunchController: UITableViewDelegate, UITableViewDataSource {
+
```

Comment:
you could also implement there the didSelectRow(at indexpath) with something like 
```swift
{
    tableView.deselectRow(at: indexPath, animated: true)
    if let cell = tableView.cellForRow(at: indexPath) as? FirstLaunchCustomPermissionCell {
        cell.toggle.isOn = !cell.toggle.isOn
        permissionValueChanged(cell.toggle)
    }
}
```

Reply:
No, I was thinking the same at first but later I realized that's against the apple standard approach. Check any items in iOS Settings. User can toggle on/off only by tapping on the toggle button.